desci-server/src/services/repoService.ts

```diff
@@ -18,8 +18,8 @@ class RepoService {
   baseUrl: string;
 
   constructor() {
-    this.#apiKey = process.env.REPO_SERVICE_SECRET_KEY;
-    this.baseUrl = process.env.REPO_SERVER_URL;
+    this.#apiKey = process.env.REPO_SERVICE_SECRET_KEY ?? '';
+    this.baseUrl = process.env.REPO_SERVER_URL ?? '';
```

Comment:
I think it's better to leave this as potentially `undefined`, otherwise we prevent the typechecker from making sure we handle that case :)